### ISSUE TITLE
Snp and aggs queries

### ISSUE THREAD

**Issue by akshala** [USCbiostats/annoq-api-v2#5]
Should there be a big single query for snp and aggs or two separate queries for them. The query parameters would be the same for both, but for histrogram in aggs bounds(max, min) and interval are needed. Should there be a separate query done within the big query to get these values for histogram? @tmushayahama @huaiyumi @mugitty 

Currently it is a single big query with initialized values for bounds(max, min) and interval.

**Comment by akshala** [comment]
Commit 0db0b3f: aggs bool added to separate aggs and snp value queries using same function

**Comment by tmushayahama** [comment]
@akshala I didn't see this until now, but I believed we discussed it offline and it would be separate query. So if you decide to add aggs_bool, maybe a better scalability approach, so as to force multiple queries is to add Enum QueryType ('aggs', 'query', 'scroll' ...) Default is query, normal search

**QueryType** 

- if aggs then the same logic with your current aggs_bool, but in addition, size=0
- if query then query normal
- if scroll, then query and return the scroll id, for custom pagination for @quemeb needs

However, we made some blunder because aggs should be a single object and query is a list. for example, a query will return a list of pos, chr, but aggs is only one value for pos and chr. So idk how to approach this. The snp model definition remains the same, Maybe separate endpoints/functions. Lemme know your thoughts
